modules/flake/overlays.nix

```diff
@@ -2,15 +2,15 @@
 {
   # Provide overlay to add `nix-snapshotter`.
   flake.overlays.default = self: super: {
-    nix-snapshotter = self.callPackage ../../package.nix {};
+    nix-snapshotter = self.callPackage ../../package.nix { };
 
-    # Depends on PR merged into main, but not yet in a release tag.
-    # See: https://github.com/containerd/containerd/pull/9028
-    containerd = super.containerd.overrideAttrs(o: {
-      src = self.fetchFromGitHub {
-        inherit (o.src) owner repo;
-        rev = "779875a057ff98e9b754371c193fe3b0c23ae7a2";
-        hash = "sha256-sXMDMX0QPbnFvRYrAP+sVFjTI9IqzOmLnmqAo8lE9pg=";
+    # Fixes https://github.com/pdtpartners/nix-snapshotter/issues/102 due to 23.11 only supporting v1.27.6 while we need v1.27.7. 
+    # Remove once we upgrade to the next stable release.
+    k3s = super.k3s.overrideAttrs (oldAttrs: {
+      k3sRepo = super.fetchgit {
+        url = "https://github.com/k3s-io/k3s";
+        rev = "v1.27.9+k3s1";
+        sha256 = "sha256-Zr9Zp9pi7S3PCTveiuSb0RebiGZrxxKC+feTAWO47Js=";
```

Comment:
@RobbieBuxton I just wanted to point out that `nixpkgs-unstable` has a recent enough version (it's currently `1.27.9+k3s1`). Should we depend on this one instead?

Reply:
Yeah, this is the same version as what unstable has and I don't think the implementation of [k3s](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/networking/cluster/k3s/builder.nix#L82) has changed since the introduction of 23.11 so this is a nice way to get an updated package without having to subscribe to two version of nixpkgs.